### PR TITLE
PAYARA-2477 Create asadmin commands for MicroProfile Metrics (Payara 5)

### PR DESCRIPTION
**Pull request by jGauravGupta**

### CODE REVIEW

**Comment by jGauravGupta** [comment]
Jenkins, quick test Payara 5 please

**Comment by jGauravGupta** [comment]
Jenkins, quick test Payara 5 please

File: appserver/payara-appserver-modules/microprofile/metrics/src/main/java/fish/payara/microprofile/metrics/admin/GetMetricsConfigurationCommand.java [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+/**
+ * AsAdmin command to get metrics configuration
+ * 
+ * @author Gaurav Gupta
+ */
+@Service(name = "get-metrics-configuration")
+@PerLookup
+@ExecuteOn({RuntimeType.DAS})
+@TargetType({DAS, STANDALONE_INSTANCE, CLUSTER, CLUSTERED_INSTANCE, CONFIG})
```
Comment (smillidge) [inline appserver/payara-appserver-modules/microprofile/metrics/src/main/java/fish/payara/microprofile/metrics/admin/GetMetricsConfigurationCommand.java]: Requires Deployment Group Target Type in Payara 5

**Comment by jGauravGupta** [comment]
Jenkins, quick test Payara 5 please

**Comment by arjantijms** [comment]
Perhaps close this and just depend on the commit for Payara 4 to be merged to 5?

**Comment by smillidge** [comment]
It needs separate PRs due to the target type of Deployment Group

File: appserver/payara-appserver-modules/microprofile/metrics/src/main/java/fish/payara/microprofile/metrics/admin/MetricsServiceConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
+/*
```
Comment (smillidge) [inline appserver/payara-appserver-modules/microprofile/metrics/src/main/java/fish/payara/microprofile/metrics/admin/MetricsServiceConfiguration.java]: Can you add an empty <metrics-service-configuration/> tag to the domain.xml for each config?

**Pull request closed without merging**